library/pod-security-policy/users/template.yaml

```diff
@@ -15,55 +15,87 @@ spec:
           properties:
             runAsUser:
               type: object
+              description: "Controls which user ID pod containers can be run with."
               properties:
                 rule:
                   type: string
+                  description: "A strategy for applying the runAsUser restriction."
+                  enum:
+                    - MustRunAs
+                    - MustRunAsNonRoot
+                    - RunAsAny
                 ranges:
                   type: array
+                  description: "A list of ranges affected by the rule."
                   items:
                     type: object
+                    description: "The range of user IDs affected by the rule."
                     properties:
                       min:
```

Comment:
docstring needed. ex: is this inclusive or exclusive?

applies to all the min/max fields in this resource.

Reply:
All inclusive.  Updated the descriptions.